src/types.rs

```diff
@@ -220,3 +220,41 @@ impl FromStr for SourceLocation {
         }
     }
 }
+
+/// Process ID.
+///
+/// # Examples
+///
+/// The default value:
+///
+/// ```
+/// use sloggers::types::ProcessID;
+///
+/// assert_eq!(ProcessID::default(), ProcessID::ProcessID);
+/// ```
+#[allow(missing_docs)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum ProcessID {
+    None,
+    ProcessID,
+}
+impl Default for ProcessID {
+    fn default() -> Self {
+        ProcessID::ProcessID
```

Comment:
I think the default value should be `ProcessID::None` to keep compatibility with past versions.

Reply:
fair enough